### PR TITLE
Replace typedef with using

### PR DESCRIPTION
**Pull request by andrei-sa**

### CODE REVIEW

File: Nexus/Include/Nexus/Definitions/Tick.hpp [hunk 1]
```diff
@@ -1,30 +1,24 @@
...
+    //! Up tick.
+    UP,
 
-      //! Down tick.
-      DOWN
-    };
-  };
-}
-
-  typedef Beam::Enumerator<Details::TickDirectionDefinition> TickDirection;
```
Comment (andrei-sa) [inline Nexus/Include/Nexus/Definitions/Tick.hpp:27]: Could not find the definition of `Beam::Enumerator` anywhere in the repository.

File: Nexus/Include/Nexus/MarketDataService/MarketOrderImbalanceExpression.hpp [hunk 2]
```diff
@@ -30,20 +30,20 @@ namespace MarketDataService {
   class MarketOrderImbalanceExpression :
       public Beam::Expressions::IterationExpression<OrderImbalance> {
```
Comment (andrei-sa) [inline Nexus/Include/Nexus/MarketDataService/MarketOrderImbalanceExpression.hpp:31]: Could not find `Beam::Expressions` either, so this file does not really compile.

File: Nexus/Include/Nexus/Definitions/Tick.hpp (hunk 1 above)
Comment (kranar) [reply Nexus/Include/Nexus/Definitions/Tick.hpp:27]: Delete this file, it's dead code.

File: Nexus/Include/Nexus/MarketDataService/MarketOrderImbalanceExpression.hpp (hunk 2 above)
Comment (kranar) [reply Nexus/Include/Nexus/MarketDataService/MarketOrderImbalanceExpression.hpp:31]: This is also dead code, delete it.

**Pull request merged**